scdf-type:
  oss:
    release: "[ERROR] [ERROR] Some problems were encountered while processing the POMs:\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n @ \n[ERROR] The build could not read 21 projects -> [Help 1]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-common-persistence:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-common/spring-cloud-dataflow-common-persistence/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-common-security-config:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-common-security-config/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-container-registry:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-container-registry/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-configuration-metadata:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-configuration-metadata/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-core-dsl:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-core-dsl/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-schema-core:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-schema-core/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-core:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-core/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-schema:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-schema/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-aggregate-task:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-aggregate-task/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-server-core:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-server-core/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-rest-resource:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-rest-resource/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-audit:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-audit/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-registry:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-registry/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-platform-kubernetes:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-platform-kubernetes/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-platform-cloudfoundry:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-platform-cloudfoundry/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-autoconfigure:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-autoconfigure/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-rest-client:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-rest-client/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-shell:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-shell/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-shell-core:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-shell-core/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-completion:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-completion/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR] \n[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.\n[ERROR] Re-run Maven using the -X switch to enable full debug logging.\n[ERROR] \n[ERROR] For more information about the errors and possible solutions, please read the following articles:\n[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException\n[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException"
    snapshot: '2.11.4-SNAPSHOT'
    milestone: '2.11.0-RC1'
    maintenance-snapshot: '2.10.4-SNAPSHOT'
  pro:
    release: '1.6.2'
    snapshot: '1.6.3-SNAPSHOT'
    maintenance-snapshot: '1.5.4-SNAPSHOT'
default:
  scdf-type: 'oss'
  version: 'release'
  package-version: "[ERROR] [ERROR] Some problems were encountered while processing the POMs:\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n[ERROR] Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16\n @ \n[ERROR] The build could not read 21 projects -> [Help 1]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-common-persistence:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-common/spring-cloud-dataflow-common-persistence/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-common-security-config:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-common-security-config/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-container-registry:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-container-registry/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-configuration-metadata:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-configuration-metadata/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-core-dsl:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-core-dsl/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-schema-core:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-schema-core/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-core:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-core/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-schema:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-schema/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-aggregate-task:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-aggregate-task/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-server-core:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-server-core/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-rest-resource:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-rest-resource/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-audit:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-audit/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-registry:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-registry/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-platform-kubernetes:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-platform-kubernetes/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-platform-cloudfoundry:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-platform-cloudfoundry/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-autoconfigure:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-autoconfigure/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-rest-client:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-rest-client/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-shell:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-shell/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-shell-core:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-shell-core/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR]   \n[ERROR]   The project org.springframework.cloud:spring-cloud-dataflow-completion:2.11.3 (/home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-completion/pom.xml) has 1 error\n[ERROR]     Non-resolvable import POM: The following artifacts could not be resolved: org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 (present, but unavailable): org.springframework.cloud:spring-cloud-deployer-dependencies:pom:2.9.3 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced @ org.springframework.cloud:spring-cloud-dataflow-parent:2.11.3, /home/runner/work/spring-cloud-dataflow/spring-cloud-dataflow/spring-cloud-dataflow-parent/pom.xml, line 397, column 16 -> [Help 2]\n[ERROR] \n[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.\n[ERROR] Re-run Maven using the -X switch to enable full debug logging.\n[ERROR] \n[ERROR] For more information about the errors and possible solutions, please read the following articles:\n[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException\n[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException"
